templates/fine-tune-llm_v2/cookbooks/customize_initial_weights_and_prompt_format/README.ipynb

```diff
@@ -0,0 +1,203 @@
+    "\n",
+    "### How do I configure access to my weights in remote storage?\n",
+    "\n",
+    "For models configured for public access, you simply need to add the URI of the location of the model weights in your training YAML. We support loading models stored on S3 (with GCS support coming soon). For private models, you could configure the read permissions for your Workspace to pull from the bucket holding your model weights. Alternatively, you could sync your model weights to your Anyscale-provided artifact storage, or even just keep it in shared storage in your workspace. The [Bring your own data](../../cookbooks/bring_your_own_data/README.md) cookbook, while focusing on datasets, provides in-depth detail on these options.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Bring checkpoints from a previous finetuning run\n",
+    "\n",
+    "This is the use case  of multi-step fine-tuning where you want to customize the base model weights to start with or the adapter weights to continue fine-tuning on the Anyscale Platform. Being able to provide the custom checkpoints for the second (or later) stage of fine-tuning is just one part of the equation. There are a number of other considerations here (What's the right order of datasets in a 2-stage fine-tuning run? How do differences in context length fit in? etc) all of which are covered in our [continue_from_checkpoint](../continue_from_checkpoint/) cookbook."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Customizing the prompt format\n",
+    "\n",
+    "## How prompt formatting works in `llmforge`\n",
+    "\n",
+    "Here's a quick rundown of how prompt formatting/ chat templating works right now: the training/validation data needs to be formatted in the OpenAI messages format. Each example has a \"messages\" entry consisting a conversation with \"system\", \"user\" and \"assistant\" roles. For each role, depending on the model, we add certain tokens as headers/footers along with a BOS token at the start of the conversation and an EOS token at the end of each assistant response. The prompt format can be specified in our YAML as a part of the `generation_config` for the model (the same format is used in our inference code):\n",
```

Comment:
```suggestion
    "Here's a quick rundown of how prompt formatting or chat templating works: the training or validation data needs to be formatted in the OpenAI messages format. Each example has a \"messages\" entry consisting a conversation with \"system\", \"user\" and \"assistant\" roles. For each role, depending on the model, we add certain tokens as headers or footers along with a BOS token at the start of the conversation and an EOS token at the end of each assistant response. The prompt format can be specified in our YAML as a part of the `generation_config` for the model (the same format is used in our inference code):\n",
```

Reply:
1. You should link to the documentation around prompt format. 
2. In the beginning of this paragraph or in another earlier paragraph you should concretize what the input format looks like via an example.